workers/tests/integrationtests/api_response_schema.py

```diff
@@ -1,42 +1,24 @@
+"""This module defines schemata for API responses.
+
+One API response should adhere to at least two schemata.
```

Comment:
Hey,
here we might write a bit easier to understand. Is the following correct:

"Every API response must adhere to the general schema. Additionally API responses need to adhere to one additional schema depending on which endpoint is used (indicator, report)."

Reply:
Yes thats correct. I used your formulation and updated the PR.